README.md

```diff
@@ -19,6 +19,7 @@ cargo install mdbook
 cargo install mdbook-admonish@1.18.0 --locked
 cargo install mdbook-katex@0.9.0 --locked
 cargo install mdbook-i18n-helpers --locked
+cargo install mdbook-variables
```

Comment:
Important: This will probably have to be added to the [`build-mdbook`](https://github.com/MystenLabs/walrus-docs/blob/ae54badb82ac68bb49c967f39779337d10675694/.github/actions/build-mdbook/action.yaml) action.

Reply:
Currently, the preview looks like this:
<img width="811" alt="image" src="https://github.com/user-attachments/assets/c3a05bac-a6ed-4889-aa66-5e61b70ec695" />